src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/application-card.component.html

```diff
@@ -1,15 +1,13 @@
-<mat-card [ngClass]="application.status === applicationStatuses.Pending ? 'card card-new' : 'card'">
-  <mat-card-content [ngClass]="userRole === role.parent ? 'card-block card-block__parent' : 'card-block'">
+<mat-card [ngClass]="application.status === ApplicationStatuses.Pending ? 'card card-new' : 'card'">
```

Comment:
class `card` is same for both conditions.
We can move it to class property, so condition will looks better.
In that case we can do `[class.card-new]="condition"` because there will be only one additional class.

Reply:
I see that you have only changed the formatting in this file, so this comment may be optional. 
But I prefer to make these changes across the file

Reply:
Unfortunately, didn't noticed the comment and merged already, but it'll come with next PR of map fixes